.env.template

```diff
@@ -7,3 +7,12 @@ FLASK_DEBUG=true
 
 # Required to use Flask's session feature (for storing todos in a browser cookie in the first exercise): https://flask.palletsprojects.com/en/2.3.x/config/#SECRET_KEY
 SECRET_KEY=secret-key
+
+#Add your TRELLO API credentials as shown below
+
+TRELLO_API_KEY=<FILL ME IN>
+TRELLO_API_TOKEN=<FILL ME IN>
+
+TRELLO_TODO_BOARD_ID=<fill me in>
+
+TRELLO_TODO_LIST_ID=<fill me in>
```

Comment:
I think we've got one entry missing here, and one mismatched variable name - worth fixing these or it does make life harder for future devs who have to track that down!
```
TRELLO_BOARD_ID=<fill me in>
TRELLO_TODO_LIST_ID=<fill me in>
TRELLO_DONE_LIST_ID=<fill me in>
```